src/communication/network_interface_cl.c

```diff
@@ -10649,6 +10649,118 @@ flashback_get_summary (dynamic_array * class_list, const char *user, time_t star
   return ER_NOT_IN_STANDALONE;
 }
 
+static char *
+unpacking_loginfo (char *ptr, int num_item)
```

Comment:
For consistency with `packing_loginfos`
```suggestion
unpacking_loginfos (char *ptr, int num_item)
```